XamarinCommunityToolkit/AssemblyInfo/AssemblyInfo.shared.cs

```diff
@@ -1,3 +1,12 @@
 ﻿using System.Runtime.CompilerServices;
+using Xamarin.Forms;
 
-[assembly: InternalsVisibleTo("Xamarin.CommunityToolkit.UnitTests")]
\ No newline at end of file
+[assembly: InternalsVisibleTo("Xamarin.CommunityToolkit.UnitTests")]
+
+#if !NETSTANDARD1_0
+[assembly: XmlnsDefinition("http://xamarin.com/schemas/2020/toolkit", "Xamarin.CommunityToolkit.Behaviors")]
+[assembly: XmlnsDefinition("http://xamarin.com/schemas/2020/toolkit", "Xamarin.CommunityToolkit.Converters")]
+[assembly: XmlnsDefinition("http://xamarin.com/schemas/2020/toolkit", "Xamarin.CommunityToolkit.Effects")]
+[assembly: XmlnsDefinition("http://xamarin.com/schemas/2020/toolkit", "Xamarin.CommunityToolkit.Extensions")]
+[assembly: XmlnsDefinition("http://xamarin.com/schemas/2020/toolkit", "Xamarin.CommunityToolkit.UI.Views")]
```

Comment:
@jfversluis Why are **Views** inside **UI**, but other aren't?

Reply:
Because they are not visible, I guess 😅

Reply:
Should we make all namespaces consistent? (With **UI** or w/out)

Reply:
if anything I would say drop the views... everything in UI.Views is a VisualElement... Behaviors, Converters, Effects, Extensions are all helpers but none of them are Visual Elements